MagneticField/GeomBuilder/src/BaseVolumeHandle.h

```diff
@@ -0,0 +1,219 @@
+#ifndef MagneticField_GeomBuilder_BaseVolumeHandle_H
+#define MagneticField_GeomBuilder_BaseVolumeHandle_H
+
+/** \class BaseVolumeHandle
+ * A temporary container to cache info on a six-surface volume during
+ * the processing. Used to sort, organise, and build shared planes.
+ * One instance is created for each DDVolume. The parameters of the 
+ * boundary surfaces are calculated during construction.
+ *
+ *  \author N. Amapane - INFN Torino
+ */
+
+#include "DataFormats/GeometrySurface/interface/Surface.h"
+#include "DataFormats/GeometrySurface/interface/ReferenceCounted.h"
+#include "DetectorDescription/Core/interface/DDSolidShapes.h"
+#include "MagneticField/VolumeGeometry/interface/VolumeSide.h"
+
+class MagVolume6Faces;
+
+namespace magneticfield {
+
+  class BaseVolumeHandle {
+  public:
+    typedef Surface::GlobalPoint GlobalPoint;
+    typedef Surface::LocalPoint LocalPoint;
+    typedef Surface::LocalVector LocalVector;
+    typedef SurfaceOrientation::GlobalFace Sides;
+
+    BaseVolumeHandle(bool debugVal = false);
+
+    // Disallow Default/copy ctor
+    // (we want to handle only pointers!!!)
+    BaseVolumeHandle(const BaseVolumeHandle& v) = delete;
+
+    virtual ~BaseVolumeHandle();
+
+    /// Return the center of the volume
+    const GlobalPoint& center() const;
+
+    /// Distance of (x,y) plane from origin
+    const double RN() const { return theRN; }
```

Comment:
While I agree this is the convention, I am not enthiusiastic to start a review of naming conventions in the package since that goes beyond the scope of the migration to dd4hep and will add even more diffs to an arleady quite extensive set of modifications.